src/test/java/seedu/address/logic/commands/DeleteAssignmentCommandTest.java

```diff
@@ -1,183 +1,129 @@
 package seedu.address.logic.commands;
 
 import org.junit.jupiter.api.Test;
+
 import seedu.address.commons.core.Messages;
 import seedu.address.commons.core.index.Index;
 import seedu.address.model.Model;
 import seedu.address.model.ModelManager;
 import seedu.address.model.UserPrefs;
+import seedu.address.model.assignment.Assignment;
 import seedu.address.model.person.Person;
 import seedu.address.testutil.PersonBuilder;
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
 import static seedu.address.testutil.TypicalIndexes.*;
+import static seedu.address.testutil.TypicalPersons.HOON;
 import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
 
 public class DeleteAssignmentCommandTest {
     private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
-    @Test
-    public void execute_validIndexUnfilteredList_success() {
-        /*Person personInList = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-        Assignment assignmentToDelete_expected =
-                TypicalAssignments.getTypicalAssignments().get(INDEX_FIRST_ASSIGNMENT.getZeroBased());
-        ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
-
-        expectedModel.addAssignment(personInList, assignmentToDelete_expected);
-        expectedModel.deleteAssignment(personInList, assignmentToDelete_expected);
-
-        personInList.getAssignments().setAssignments(TypicalAssignments.getTypicalAssignments());
-        model.updateFilteredAssignmentList(personInList);
-        Assignment assignmentToDelete_model =
-                model.getFilteredAssignmentList().get(INDEX_FIRST_ASSIGNMENT.getZeroBased());
-        DeleteAssignmentCommand deleteAssignmentCommand =
-                new DeleteAssignmentCommand(personInList.getName(), INDEX_FIRST_ASSIGNMENT);
-
-        String expectedMessage =
-                String.format(DeleteAssignmentCommand.MESSAGE_DELETE_ASSIGNMENT_SUCCESS, assignmentToDelete_model);
+    private Person clonePerson(Person person) {
+        return new PersonBuilder(person).build();
+    }
 
-        assertCommandSuccess(deleteAssignmentCommand, model, expectedMessage, expectedModel);*/
+    private Model setUpModel(Model inputModel, Person selectedPerson, Person clonedPerson) {
+        inputModel.setPerson(selectedPerson, clonedPerson);
+        inputModel.setPerson(selectedPerson, clonedPerson);
```

Comment:
two repeated?

Reply:
also, can move to commandTestUtil class, since mark assignment face the same issue, also need to be used in mark command

Reply:
Good point. Since you have shifted to ```CommandTestUtil```, I have deleted these methods in ```deleteAssignmentTest``` and imported from there instead. 

Also, I have refactored and changed the name of the  static methods for ```setUpActualModel``` to ```replacePersonWithClone``` and ```setUpExpectedModel``` to ```setUpNewModelWithClonedPerson``` to reflect what the methods do more accurately.